x-pack/plugins/security_solution/server/lib/entity_analytics/asset_criticality/check_and_init_asset_criticality_resources.ts

```diff
@@ -30,8 +30,9 @@ export const checkAndInitAssetCriticalityResources = async (
   });
 
   const doesIndexExist = await assetCriticalityDataClient.doesIndexExist();
+  const isIndexMappingOutdated = await assetCriticalityDataClient.isIndexMappingsOutdated();
```

Comment:
I would prefer only to do this check when a version is updated, but we do not store the asset criticality version anywhere.